### PR TITLE
♿ Added accessibility name to play pause button

### LINKED ISSUE #30221: amp-story: Play/pause button is unlabeled (A11Y)
Screen Shot 2020-09-14 at 12 27 44 PM

### PR DESCRIPTION
**Pull request by mszylkowski**
Closes #30221

### CODE REVIEW

**Review by newmuis — COMMENTED** [review]
Can you also add to `_locales/default.js` with maybe just "play" and "pause" as the strings?

File: extensions/amp-story/1.0/_locales/en.js [hunk 1]
```diff
@@ -207,6 +207,14 @@ const strings = {
     description:
       'Label for the fourth answer choice from a multiple choice quiz (e.g. D in A/B/C/D)',
   },
+  [LocalizedStringId.AMP_STORY_PAUSE_BUTTON_LABEL]: {
+    string: 'Pause story',
+    description: 'Label for the pause button on the system layer',
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/_locales/en.js:212]: “Without knowing the context of a message, a translator can't do much better than machine translation. Writing good message descriptions will help the translator understand the purpose of the message and its relationship to the surrounding messages.
[...]
Before writing a description, you should realize that translators will not know the product as well as you do. They work on multiple products, and they may not be engineers, so you should write descriptions that are understandable by a general audience.”

Translators won't know what a Story or a system layer is. I'm pinging you a link to good/bad examples.

**Comment by mszylkowski** [comment]
The other buttons from the system layer are also missing from default.js (share button label, audio labels), should I add them all?

File: src/localized-strings.js [hunk 2]
```diff
@@ -59,6 +59,8 @@ export const LocalizedStringId = {
   AMP_STORY_PAGINATION_BUTTON_PREVIOUS_PAGE_LABEL: '82',
   AMP_STORY_PAGE_ERROR_VIDEO: '65',
   AMP_STORY_PAGE_PLAY_VIDEO: '34',
+  AMP_STORY_PAUSE_BUTTON_LABEL: '85',
```
Comment (gmajoulet) [inline src/localized-strings.js:62]: Please update the `Next ID`

Also, shouldn't this one be `84`?

**Review by gmajoulet — APPROVED** [review]

File: src/localized-strings.js (hunk 2 above)
Comment (mszylkowski) [reply src/localized-strings.js:62]: 84 is the "SCORE:" from the results component, but I also forgot to update that

**Comment by gmajoulet** [comment]
cc @choumx for `[NEEDS APPROVAL] src/localized-strings.js (requested: choumx)`

**Review by choumx — APPROVED** [review]

**Pull request merged**